File: src/java/com/entity/ReviewBook.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.entity;

import java.util.Date;

/**
 *
 * @author Administrator
 */
public class ReviewBook {

    private int id;
    private int idBook;
    private int idUser;
    private String content;
    private Date CreateDate;
    private String username;

    public ReviewBook() {

    }

    public ReviewBook(int id, int idBook, int idUser, String content, Date CreateDate) {
        this.id = id;
        this.idBook = idBook;
        this.idUser = idUser;
        this.content = content;
        this.CreateDate = CreateDate;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getIdBook() {
        return idBook;
    }

    public void setIdBook(int idBook) {
        this.idBook = idBook;
    }

    public int getIdUser() {
        return idUser;
    }

    public void setIdUser(int idUser) {
        this.idUser = idUser;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public Date getCreateDate() {
        return CreateDate;
    }

    public void setCreateDate(Date CreateDate) {
        this.CreateDate = CreateDate;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

}

File: web/js/review_book.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

$(document).ready(function () {
    getReviewByTop(document.getElementById("idBook").value, 10);
});

function getReviewByTop(idBook, top) {
    // delete all current review
    var main = document.getElementById("content-article");
    while (main.hasChildNodes()) {
        main.removeChild(main.lastChild);
    }
    // get data review
    $.ajax({
        type: "GET",
        url: "ReviewBookController",
        data: {idBook: idBook, top: top},
        contentType: "application/json;charset=utf-8",
        dataType: "json",
        success: function (result) {
            var main = document.getElementById("content-article");
            for (var i in result) {
                var article = document.createElement("article");
                var a = document.createElement("a");
                a.href = "ProfileController?idUser=" + result[i].idUser;
                a.innerHTML = result[i].username;
                var p = document.createElement("p");
                p.innerHTML = result[i].createDate;
                var textarea = document.createElement("textarea");
                textarea.readonly = true;
                textarea.value = result[i].content;
                article.appendChild(a);
                article.appendChild(p);
                article.appendChild(textarea);
                main.appendChild(article);
            }
        },
        error: function (response) {
            alert('eror');
        }
    });
}

function getSelectTop() {
    var value = document.getElementById("select-top");
    getReviewByTop(document.getElementById("idBook").value, value.options[value.selectedIndex].value);
}

document.getElementById("sendReivew").addEventListener("submit", function (event) {
    event.preventDefault(); // set form do not reload when submit
    $.ajax({
        url: 'ReviewBookController',
        type: 'post',
        data: {
            idBook: document.getElementById("idBook").value,
            content: document.getElementById("content-review").value
        },
        success: function (data) {
            if (data === "true") {
                document.getElementById("content-review").value = "";
                getSelectTop();
                setTimeout(function () {
                    $('html, body').animate({
                        scrollTop: $("div.scrollhere").offset().top
                    }, 1000);
                }, 100);
            }
        },
        error: function () {
            alert('error');
        }
    });



});
